### ISSUE TITLE
libclc new release

### ISSUE THREAD

**Issue by kloczek** [llvm/llvm-project#188]
Looks like Mesa needs on build libclc. At the same time llvm libclc looks like never have been released (cannot find any dist tar ball).

Is it any plan to make release of the libclc?

**Comment by awatry** [comment]
Hi Tomasz,

You're right that Mesa does build against libclc, if the Clover state
tracker is enabled. Unless something has changed, that's an optional
dependency.  Most distributions I've checked have been building snapshots
of libclc and shipping those, but it would be nice to get a release of
libclc out so that people aren't using random git snapshots.

That said, we need to determine what the criteria for a 1.0 release of
libclc would be.  I had been hoping that we'd be able to get the Clover
implementation through the OpenCL 1.2 conformance test suite successfully
for at least one hardware/software combination, at which point I'd be
enthusiastic about cutting a release.

Right now, from a libclc standpoint, I think we're close.  We still need a
printf() implementation, and there's a few simple functions that might be
missing (don't remember if we merged clz/popcount implementations or punted
on it), but what is implemented so far I believe is reliable.

We *may* be to the point that we'd pass a CL 1.1 conformance run (based on
the percentage of the libclc-focused 1.2 tests we do pass), but since
there's no public CL 1.1 CTS code, I can't actually officially test it,
hence trying to get to 1.2 for which there is an available test suite.

--Aaron
